src/modules/Dashboard/components/TakeNowModal/TakeNowModal.tsx

```diff
@@ -294,7 +294,21 @@ export const useTakeNowModal = ({ dataTestId }: UseTakeNowModalProps) => {
         });
 
         setActivityOrFlow(null);
-        window.open(url.toString(), '_blank');
+
+        const newTab = window.open(url.toString(), '_blank');
+        // message received from the TakeNowSuccessModal.tsx file from the web app
+        // it's needed to close the new tab after the user clicks the "Close" button
```

Comment:
Nit: suggested minor improvement in comment clarity 😇 
```suggestion
        // Listen for message received from Web App's TakeNowSuccessModal component when it
        // requests to close its tab when the "Return to Admin App" button is clicked.
```